addshare/addshare.c

```diff
@@ -207,9 +209,12 @@ int main(int argc, char *argv[])
 		ret = command_update_share(smbconf, arg_name, arg_opts);
 
 	if (cmd && !ret) {
-		ret = send_signal_to_ksmbd_mountd(SIGHUP);
-		if (ret)
+		int tmp_log_level = log_level <= PR_INFO ? PR_NONE : log_level;
+
+		toggle_log_level(tmp_log_level);
+		if (send_signal_to_ksmbd_mountd(SIGHUP))
 			pr_err("Failed to notify ksmbd.mountd of changes\n");
```

Comment:
if notifying is failed In the situation where mountd runs, we don't need to let users know the failure?

Reply:
@hclee, please see the discussion at https://github.com/cifsd-team/ksmbd-tools/pull/271#issuecomment-1214161793 and onwards. I don't personally think there is a need to hide these error prints but users may be confused by them.